backend/jobs/metric-alerts/metric-alerts.go

```diff
@@ -27,6 +27,10 @@ import (
 const maxWorkers = 40
 const alertEvalFreq = time.Minute
 
+var defaultAlertFilters = map[modelInputs.ProductType]string{
+	modelInputs.ProductTypeErrors: "status=OPEN ",
```

Comment:
wonder what the expectation is with folks using alerting, perhaps something we can add to the UI to make it clear that error alerts by default only trigger for open errors?

Reply:
I can add that to the info - I think this does keep it in line with the old alerts: https://github.com/highlight/highlight/blob/main/backend/public-graph/graph/resolver.go#L1798